index.html

```diff
@@ -499,511 +355,1145 @@ <h3>Multikey</h3>
-      </li>
-    </ul>
-    <p>This operation can be applied by any 
-      <a data-cite="VC-DATA-MODEL-2#dfn-holders">holder</a> of a `bbs-signature-2023`
-      secured document, and as such, `bbs-proof-2023` MUST be implemented with
-      awareness of the mandatory to disclose fields the original 
-      <a data-cite="VC-DATA-MODEL-2#dfn-issuers">issuer</a> required.
-    </p>
-    <p class="issue" data-number="3"></p>
-    <section>
-      <h3>Add Proof </h3>
-      <p class="issue" data-number="81"></p>
-    </section>
-    <section>
-      <h3>Verify Proof </h3>
-      <p class="issue" data-number="82"></p>
-    </section>
-    <section>
-      <h3>Transformation </h3>
-      <p class="issue" data-number="83"></p>
-    </section>
-    <section>
-      <h3>Hashing </h3>
-      <p class="issue" data-number="84"></p>
-    </section>
-    <section>
-      <h3>Proof Configuration </h3>
-      <p class="issue" data-number="85"></p>
-    </section>
+
+
+
     <section>
-      <h3>Proof Serialization </h3>
-      <p class="issue" data-number="85"></p>
+      <h2>Privacy Considerations</h2>
+      <p class="issue">TODO: We need to add a complete list of privacy
+      considerations.</p>
     </section>
+
     <section>
-      <h3>Proof Verification </h3>
-      <p class="issue" data-number="85"></p>
+      <h2>Security Considerations</h2>
+      <p class="issue">TODO: We need to add a complete list of security
+      considerations.</p>
     </section>
-      
-  </section>
-</section>
-
-
-
-<section>
-  <h2>Privacy Considerations</h2>
-  <p class="issue">TODO: We need to add a complete list of privacy
-  considerations.</p>
-</section>
+    <section class="appendix informative">
+      <h2>Test Vectors</h2>
+      <p>
+To demonstrate selective disclosure features including mandatory disclosure,
+selective disclosure, and overlap between mandatory and selective disclosure
+requires an input credential document with more content than previous test
+vectors. To avoid excessively long test vectors the starting document test
+vector is based on a purely ficticious windsurfing (sailing) competition
+scenario. In addition we break the test vectors into two groups based on those
+that would be generated by the issuer (base proof) and those that would be
+generated by the holder (derived proof).
+        </p>
+        <section>
+          <h4>Base Proof</h4>
+          <p>
+In order to add a selective disclosure base proof to a document the issuer needs
+the following cryptographic key material:
+          </p>
+          <ol>
+            <li>
+The issuers private/public key pair, i.e., the key pair corresponding to the
+verification method that will be part of the proof.
+            </li>
+            <li>
+An HMAC key. This used to randomize the order of the blank node ids to avoid
+potential information leakage from the blank node id ordering. This is used only
+once and is shared between issuer and holder. The HMAC in this case is
```

Comment:
```suggestion
An HMAC key. This is used to randomize the order of the blank node IDs to avoid
potential information leakage via the blank node ID ordering. This is used only
once, and is shared between issuer and holder. The HMAC in this case is
```